cmd/veneur-emit/main.go

```diff
@@ -469,6 +471,11 @@ func createMetric(span *ssf.SSFSpan, passedFlags map[string]flag.Value, name str
 			}
 			span.Metrics = append(span.Metrics, ssf.Count(name, float32(value), tags))
 		}
+
+		if passedFlags["set"] != nil {
+			logrus.Debugf("Sending set '%s' -> %s", name, passedFlags["set"].String())
+			span.Metrics = append(span.Metrics, ssf.Set(name, passedFlags["set"].String(), tags))
```

Comment:
Do we need to protect against `""` here?

Reply:
I don't think so - a `""` value would mean that `-ssf` had been passed (it's the magic of that `passedFlags` structure), and should count as a unique value (: